frontend/src/app/user/user.component.html

```diff
@@ -1 +1,38 @@
-<p>user works!</p>
+<div class="jumbotron">
+  <div class="container-fluid">
+    <div *ngIf="userInfo; else userError" class="container">
+      <div class="row">
+        <div class="col">
+          <img
+            class="rounded-circle"
+            src="../../assets/images/user/232960.jpeg"
+            width="250"
```

Comment:
Personally I like to have all the styles in the CSS, but I see why in this case it could be done directly in the HTML.

Reply:
Absolutely, will transfer the styles to a css file